src/Documentation/doc/locales/zh-CN/help/getting_started.md

```diff
@@ -0,0 +1,766 @@
+
+`hack()` / `grow()` / `weaken()` 需要使用 await 关键字，因为它们与其他命令不同，需要一定的时间才能完成执行。
+如果你忘记 await 这些命令，那就会产生一个异常来告知你试图同时做多件事。这是因为你的代码会立即完成函数调用，而不会等待操作完成。
+同样重要的是，await 只能在标记为 `async` 的函数中使用(注意：`main()` 已被标记为 `async`)
+
+## 运行我们的脚本
+
+现在，我们要开始运行我们的[黑客攻击](../basic/hacking.md)[脚本](../basic/scripts.md)，这样它就能开始为我们赚钱和积累经验了。
+我们的主机只有 8GB [内存](../basic/ram.md)，我们稍后会交给它一些别的任务。
+所以，我们将利用其他机器上的[内存](../basic/ram.md)来完成这个工作。
+
+前往 `终端 \ Terminal` 并输入以下命令：
+
+    $ scan-analyze 2
+
+这将显示网络上某些[服务器](../basic/servers.md)的详细信息。
+**网络是随机的，因此每个人的输出都不同**。
+以下是我制作它时显示的内容：
+
+    [home ~]> scan-analyze 2
+    ┕ home
+      ┃   Root Access: YES, Required hacking skill: 1
+      ┃   Number of open ports required to NUKE: 5
+      ┃   RAM: 8.00GB
+      ┣ n00dles
+      ┃ ┃   Root Access: YES, Required hacking skill: 1
+      ┃ ┃   Number of open ports required to NUKE: 0
+      ┃ ┃   RAM: 4.00GB
+      ┃ ┕ nectar-net
+      ┃       Root Access: NO, Required hacking skill: 20
+      ┃       Number of open ports required to NUKE: 0
+      ┃       RAM: 16.00GB
+      ┣ foodnstuff
+      ┃ ┃   Root Access: NO, Required hacking skill: 1
+      ┃ ┃   Number of open ports required to NUKE: 0
+      ┃ ┃   RAM: 16.00GB
+      ┃ ┕ zer0
+      ┃       Root Access: NO, Required hacking skill: 75
+      ┃       Number of open ports required to NUKE: 1
+      ┃       RAM: 32.00GB
+      ┣ sigma-cosmetics
+      ┃ ┃   Root Access: NO, Required hacking skill: 5
+      ┃ ┃   Number of open ports required to NUKE: 0
+      ┃ ┃   RAM: 16.00GB
+      ┃ ┕ max-hardware
+      ┃       Root Access: NO, Required hacking skill: 80
+      ┃       Number of open ports required to NUKE: 1
+      ┃       RAM: 32.00GB
+      ┣ joesguns
+      ┃     Root Access: NO, Required hacking skill: 10
+      ┃     Number of open ports required to NUKE: 0
+      ┃     RAM: 16.00GB
+      ┣ hong-fang-tea
+      ┃     Root Access: NO, Required hacking skill: 30
+      ┃     Number of open ports required to NUKE: 0
+      ┃     RAM: 16.00GB
+      ┣ harakiri-sushi
+      ┃     Root Access: NO, Required hacking skill: 40
+      ┃     Number of open ports required to NUKE: 0
+      ┃     RAM: 16.00GB
+      ┕ iron-gym
+        ┃   Root Access: NO, Required hacking skill: 100
+        ┃   Number of open ports required to NUKE: 1
+        ┃   RAM: 32.00GB
+        ┕ CSEC
+              Root Access: NO, Required hacking skill: 55
+              Number of open ports required to NUKE: 1
+              RAM: 8.00GB
+
+记下这些服务器：
+
+- `sigma-cosmetics`
+- `joesguns`
+- `nectar-net`
+- `hong-fang-tea`
+- `harakiri-sushi`
+
+上述所有服务器都拥有 16GB[内存](../basic/ram.md)。
+此外，它们都不需要任何开放端口即可进行 NUKE。
+换句话说，我们可以获得这些服务器的 root 权限，然后在它们上面运行[脚本](../basic/scripts.md)。
+
+首先，让我们确定可以运行多少个线程的[黑客攻击](../basic/hacking.md)[脚本](../basic/scripts.md)。
+(有关多线程的更多信息，请参阅[脚本](../basic/scripts.md)页面)
+
+我们写的这个[脚本](../basic/scripts.md)需要使用 2.6GB 的[内存](../basic/ram.md)。
+你可以通过下面的 `终端 \ Terminal` 命令来查看：
```

Comment:
```suggestion
你可以通过下面的 `终端 / Terminal` 命令来查看：
```
Later on, you use forward-slash for showing translated + literal together, probably best to be consistent.

Reply:
Yes, I was going to use / for `translated content / original text`. The underlying reason for this is that the UI is not translated, so translating the entire literal content could be confusing.
But the document was so long that it took me over a day to complete it and make sure that the translated descriptions fit the Chinese language as well as possible and that the formatting was consistent. At the same time, I was eager to present a draft to see if it was feasible.
The problem is probably the result of some typing error with Ctrl-C & Ctrl-V. I'll fix it.

Reply:
@loliconRoot 
how can i translate to pt-BR, where can i find the file, can i?

Reply:
@FilhodoSenhor This PR was never finished/submitted. You can do the same thing they were doing here, but I'll give you the same warning up-front: It'll be a lot of work.